### PR TITLE
feature/331

### LINKED ISSUE #331: New Airport: KELP
El Paso International Airport (IATA: ELP, ICAO: KELP, FAA LID: ELP) is a public airport four miles (6 km) northeast of downtown El Paso, in El Paso County, Texas, US. It is the largest commercial airport in West Texas, handling 2,778,248 passengers in 2014. The airport serves the El Paso-Las Cruces CSA.

### PR DESCRIPTION
**Pull request by Fechulo**
Resolves #331.

### Completion Checklist
- [x] terrain file
- [x] polygonal airspace boundary
- [x] all SIDs
- [x] all STARs
- [x] information to validate traffic levels
- [x] base video map
- [x] no console errors or warnings

Add El Paso Int'l Airport.

---
According to Wikipedia, the airport has 79,148 yearly operations. Traffic level should fall between 9acph and 27.1acph. Currently, there's a total of 20acph 👍

### CODE REVIEW

**Comment by Fechulo** [comment]
Now that f/327 is merged, this one is ready for review!

File: assets/airports/kelp.json [hunk 1]
```diff
@@ -0,0 +1,3588 @@
...
+                "TXSAS": ["TXSAS", "TELRE", "DLINE"]
+            },
+            "body": [["UUTEP", "A120+|A140-"], "MYNRS", ["BEAHR", "A80|S210"]],
+            "rwy": {
+                "4": [],
+                "8R": [],
+                "22": [],
+                "26L": [],
+                "26R": []
+            }
```
Comment (Fechulo) [inline assets/airports/kelp.json]: Done 👍

**Comment by Fechulo** [comment]
I don't even know how that happened... :man_facepalming: Done now!

**Comment by Fechulo** [comment]
I'd like to re-do the terrain for this one.

**Comment by Fechulo** [comment]
(Yes, I'm going to keep uploading before/after pictures, they look awesome)
Before:
elp before
After:
elp after

File: assets/airports/kelp.json [hunk 2]
```diff
@@ -0,0 +1,3606 @@
...
+            ],
+            "ils": [false, false]
+        }
+    ],
+    "sids": {
+        "ATKNN5": {
+            "icao": "ATKNN5",
+            "name": "Atkin Five",
+            "rwy": {
+                "8R": [["^_RWY26L04.08DME", "A60+|S220-"], ["JAPOD", "S220-"]],
```
Comment (erikquinn) [inline assets/airports/kelp.json]: No speed restriction until `JAPOD`

File: assets/airports/kelp.json [hunk 3]
```diff
@@ -0,0 +1,3606 @@
...
+                ["ATKNN", "HAVID", "ALIBY", "NOCHI*"],
+                ["FEPOX", "BOATY", "SLNNK*"],
+                ["ATKNN", "ZILEK", "FLOGS*"]
+            ]
+        },
+        "JCOXX2": {
+            "icao": "JCOXX2",
+            "name": "Jaycox Two",
+            "rwy": {
+                "4": [["^_RWY2204.98DME", "A65+|S230-"]]
```
Comment (erikquinn) [inline assets/airports/kelp.json]: No speed restriction here

File: assets/airports/kelp.json [hunk 4]
```diff
@@ -0,0 +1,3606 @@
...
+                ["SLNNK*"],
+                ["GREBE*"]
+            ]
+        },
+        "LATVE3": {
+            "icao": "LATVE3",
+            "name": "Latevee Three",
+            "rwy": {
+                "4": ["KERGE", "DOTGE"],
+                "8R": ["MATLE", ["CATEG", "A110+|S250"]],
```
Comment (erikquinn) [inline assets/airports/kelp.json]: No speed restriction here

File: assets/airports/kelp.json (hunk 2 above)
Comment (Fechulo) [reply assets/airports/kelp.json]: done 👍

File: assets/airports/kelp.json (hunk 3 above)
Comment (erikquinn) [reply assets/airports/kelp.json]: touche :+1:

File: assets/airports/kelp.json (hunk 4 above)
Comment (erikquinn) [reply assets/airports/kelp.json]: touche :+1:

**Pull request merged**